### PR TITLE
Addition of detection timestamp, severity, and score to the CVEs alerts inventory for Vulnerability Detector - Vulnerability Detector Integration Tests Implementation

### PR DESCRIPTION
**Pull request by pereyra-m**
|Related issue|
|---|
|#1461|

## Description

This PR adds all the necessary changes to the  **scan_results** tests to verify the correct vulnerabilities score and severity after a vulnerability detector scan completion. These new fields were added in the epic [8727](https://github.com/wazuh/wazuh/issues/8727).

## Tests

- [x] Proven that tests **pass** when they have to pass.

- [x] Python codebase satisfies PEP-8 style style guide. `pycodestyle --max-line-length=120 --show-source --show-pep8 file.py`.
- [x] Python codebase is documented following the Google Style for Python docstrings.
- [x] The test is documented in wazuh-qa/docs.
- [x] `provision_documentation.sh` generate the docs without errors.

### CODE REVIEW

**Comment by pereyra-m** [comment]
### Jenkins execution

https://ci.wazuh.info/job/Test_integration/6590/

sucess_IT_improvements

**Comment by MiguelazoDS** [comment]
# Tests after rebase

## Test Windows

image

## Test General Settings
__Some tests fails randomly. But this also occurs in the `1261-cves-alerts-inventory` branch.__
image

## Test Scan Types

image

## Test Scan Results

image

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_archlinux_inventory.yaml [hunk 1]
```diff
@@ -34,22 +34,4 @@
...
-    - disabled:
-        value: 'yes'
-  - section: wodle
-    attributes:
-      - name: 'syscollector'
-    elements:
-      - disabled:
-          value: 'yes'
\ No newline at end of file
+            value: '5s'
```
Comment (pereyra-m) [inline tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_archlinux_inventory.yaml:37]: Maybe we should keep these modules turned off, to improve the stability of the test

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_debian_inventory.yaml [hunk 2]
```diff
@@ -25,21 +25,3 @@
             value: NVD_JSON_PATH
         - update_interval:
             value: '5s'
```
Comment (pereyra-m) [inline tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_debian_inventory.yaml:27]: Maybe we should keep these modules turned off, to improve the stability of the test

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_macos_inventory.yaml [hunk 3]
```diff
@@ -24,22 +24,4 @@
...
-    - disabled:
-        value: 'yes'
-  - section: wodle
-    attributes:
-      - name: 'syscollector'
-    elements:
-      - disabled:
-          value: 'yes'
\ No newline at end of file
+            value: '5s'
```
Comment (pereyra-m) [inline tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_macos_inventory.yaml:27]: Maybe we should keep these modules turned off, to improve the stability of the test

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_msu_inventory.yaml [hunk 4]
```diff
@@ -35,21 +35,3 @@
             value: NVD_JSON_PATH
         - update_interval:
             value: '5s'
-  - section: sca
```
Comment (pereyra-m) [inline tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_msu_inventory.yaml:38]: Maybe we should keep these modules turned off, to improve the stability of the test

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_provider_and_nvd_configuration.yaml [hunk 5]
```diff
@@ -24,22 +24,4 @@
...
-    - disabled:
-        value: 'yes'
-  - section: wodle
-    attributes:
-      - name: 'syscollector'
-    elements:
-      - disabled:
-          value: 'yes'
\ No newline at end of file
+            value: '5s'
```
Comment (pereyra-m) [inline tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_provider_and_nvd_configuration.yaml:27]: Maybe we should keep these modules turned off, to improve the stability of the test

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_redhat_inventory.yaml [hunk 6]
```diff
@@ -31,21 +31,3 @@
             value: NVD_JSON_PATH
         - update_interval:
             value: '5s'
```
Comment (pereyra-m) [inline tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_redhat_inventory.yaml:33]: Maybe we should keep these modules turned off, to improve the stability of the test

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_ubuntu_inventory.yaml [hunk 7]
```diff
@@ -25,21 +25,3 @@
             value: NVD_JSON_PATH
         - update_interval:
             value: '5s'
-  - section: sca
```
Comment (pereyra-m) [inline tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_ubuntu_inventory.yaml:28]: Maybe we should keep these modules turned off, to improve the stability of the test

**Review by pereyra-m — COMMENTED** [review]
Only some comments to discuss with the team

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_debian_inventory.yaml (hunk 2 above)
Comment (MiguelazoDS) [reply tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_debian_inventory.yaml]: You're right. Fixed

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_macos_inventory.yaml (hunk 3 above)
Comment (MiguelazoDS) [reply tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_macos_inventory.yaml]: Fixed

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_msu_inventory.yaml (hunk 4 above)
Comment (MiguelazoDS) [reply tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_msu_inventory.yaml]: Fixed

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_provider_and_nvd_configuration.yaml (hunk 5 above)
Comment (MiguelazoDS) [reply tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_provider_and_nvd_configuration.yaml]: Fixed

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_redhat_inventory.yaml (hunk 6 above)
Comment (MiguelazoDS) [reply tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_redhat_inventory.yaml]: Fixed

File: tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_ubuntu_inventory.yaml (hunk 7 above)
Comment (MiguelazoDS) [reply tests/integration/test_vulnerability_detector/test_scan_results/data/wazuh_ubuntu_inventory.yaml]: Fixed

**Comment by DProvinciani** [comment]
As the epic https://github.com/wazuh/wazuh/issues/8727 is just an enhancement over the epic https://github.com/wazuh/wazuh/issues/7749, we decided to merge all the 8727 related PRs into the 7749 branches and finalize all the remaining QA work over the epic https://github.com/wazuh/wazuh/issues/7749.

**Pull request merged**